Tasks/Banchtchikov/Task1/Task1.py

```diff
@@ -0,0 +1,15 @@
+deposit = int(input('Enter deposit amount:\n'))
+rate = float(input('Enter interest rate\n'))
+n = int(input('Enter the number of years\n'))
```

Comment:
it's a money-related task, so I advise to use the Decimal data type for the calculation, -1 point